TreeSearch.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Scanner;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class TreeSearch {
```

Comment:
first of all, thank you so much for only giving me the BFS code without other code of Magpie, Princeton, or morse code that I don't want to look at right now!